tee/prototype/epochproc.go

```diff
@@ -120,9 +124,12 @@ func (e *Enclave) txRoutine(
 			})
 			log.Debug("txRoutine: TX received, applying...")
 			stagedTxs.cacheTx(tx)
-			err := e.applyEpochTx(txEpoch, tx)
-			if err != nil {
+			if err := e.applyEpochTx(txEpoch, tx); err != nil {
```

Comment:
The whole enclave design is not production ready as it is right now anyways ;) it's just still a demo, after all. I propose to discuss fundamental changes to how we deal with invalid txs in a new issue. This PR should only provide one thing, error/result receipts to the caller of `ProcessTXs`/`ProcessBlocks`.

(Pre-validation has the big disadvantage that we'd check all signatures twice on the same machine, since the op would share computational ressorces with the TEE. In a production setting this could be solved by a layered approach, where one part of the operator is running on a different machine, just receiving and validating txs. But this only becomes relevant in the far future, I guess ;) )